appcues/src/main/java/com/appcues/trait/appcues/TooltipTrait.kt

```diff
@@ -387,12 +387,13 @@ internal class TooltipTrait(
         windowInfo: AppcuesWindowInfo,
         tooltipSettings: TooltipSettings,
         useBottomToastStyle: Boolean,
+        maxHeight: Dp?,
     ): Modifier =
         composed {
             // keep containerMaxSize here as a rememberable so it force recomposition when value change
             val containerMaxSize = DpSize(
                 width = windowInfo.widthDp - (SCREEN_HORIZONTAL_PADDING * 2),
-                height = windowInfo.heightDp - (SCREEN_VERTICAL_PADDING * 2)
+                height = maxHeight ?: (windowInfo.heightDp - (SCREEN_VERTICAL_PADDING * 2))
```

Comment:
maxHeight coming from tooltipPosition, if null we use previous value (screen size - paddings)